src/app/item-popup/DesktopItemActions.m.scss

```diff
@@ -142,3 +123,59 @@
     transform: scaleX(-1);
   }
 }
+
+.moveLocations {
+  flex-direction: column;
+  align-items: inherit;
+  justify-self: flex-end;
+  padding: 6px 12px 0;
+}
+
+.moveLocationIcons {
+  display: flex;
+  flex-direction: row;
+  justify-content: space-between;
+  margin: 6px 0;
+}
+
+.move,
+.equip {
+  cursor: pointer;
+  position: relative;
+  display: flex;
+  align-content: center;
+  justify-content: center;
+  height: 36px;
+  width: 36px;
+
+  img {
+    position: absolute;
+    height: 100%;
+    width: 100%;
+    top: 0;
+    left: 0;
+  }
+
+  :global(.app-icon) {
+    position: absolute;
+    height: 60%;
+    width: 60%;
+    top: 7px;
+    left: 7px;
+    filter: drop-shadow(1px 1px 2px black);
+  }
+
+  &:hover {
+    filter: brightness(200%);
```

Comment:
Do we need both the brightness and the box? For the character headers I ditched the brightness

Reply:
was trying to mimic what they were doing in game on hover, need to actually see what they're doing instead of guessing. can try another time- removed for now.